superset/assets/visualizations/nvd3_vis.js

```diff
@@ -173,6 +177,16 @@ function nvd3Vis(slice, payload) {
         } else {
           chart = nv.models.lineChart();
         }
+
+        if (fd.line_fill_area) {
+          data = data.map(d => ({
+            ...d,
+            area: true,
```

Comment:
Looked at the `nvd3` source code and looks like you can pass `fillOpacity: 0.05` right here under `area`